### ISSUE TITLE
IllegalArgumentException: Argument not in an array

### ISSUE THREAD

**Issue by Nikitakhanna92** [doanduyhai/Achilles#277]
Hi @doanduyhai 
I got an exception IllegalArgumentException: Argument not in an array while creating List<Integer> in a UDT. The table was successfully created but got this error while inserting data and if I try to insert data from cql using insert command then it worked succesfully.
```
@UDT(name = "targeting")
public class Targeting {

  @Column
  private List<Integer> includeIds;

  public List<Integer> getIncludeIds() {
    return includeIds;
  }

  public void setIncludeIds(List<Integer> includeIds) {
    this.includeIds = includeIds;
  }
}

@Table
public class Dummy {

@Column
 private Targeting target;

public Targeting getTarget() {
return target;
}

public void setTarget(Targeting target) {
this.target = target;
}

}
```
CQL QUERY:
```
INSERT INTO dummy (target) VALUES ({includeIds:[425, 569]});
```
 Exception:
```
java.lang.IllegalArgumentException: Argument is not an array
    at java.lang.reflect.Array.getLength(Native Method)
    at com.datastax.driver.extras.codecs.arrays.AbstractPrimitiveArrayCodec.serialize(AbstractPrimitiveArrayCodec.java:52)
    at com.datastax.driver.core.AbstractData.setList(AbstractData.java:358)
    at com.datastax.driver.core.AbstractData.setList(AbstractData.java:374)
    at info.archinnov.achilles.internals.metamodel.ListProperty.encodeFieldToUdt(ListProperty.java:184)
    at info.archinnov.achilles.generated.meta.udt.GeoTargeting_AchillesMeta.createUDTFromBean(GeoTargeting_AchillesMeta.java:128)
    at info.archinnov.achilles.generated.meta.udt.GeoTargeting_AchillesMeta.createUDTFromBean(GeoTargeting_AchillesMeta.java:1)
    at info.archinnov.achilles.internals.metamodel.UDTProperty.encodeFromJavaInternal(UDTProperty.java:85)
    at info.archinnov.achilles.internals.metamodel.UDTProperty.encodeFromJavaInternal(UDTProperty.java:51)
    at info.archinnov.achilles.internals.metamodel.AbstractProperty.encodeFromJava(AbstractProperty.java:57)
    at info.archinnov.achilles.internals.metamodel.AbstractProperty.encodeField(AbstractProperty.java:90)
    at info.archinnov.achilles.internals.metamodel.UDTProperty.encodeFieldToUdt(UDTProperty.java:128)
    at info.archinnov.achilles.generated.meta.udt.OfferTargeting_AchillesMeta.createUDTFromBean(OfferTargeting_AchillesMeta.java:109)
    at info.archinnov.achilles.generated.meta.udt.OfferTargeting_AchillesMeta.createUDTFromBean(OfferTargeting_AchillesMeta.java:1)
    at info.archinnov.achilles.internals.metamodel.UDTProperty.encodeFromJavaInternal(UDTProperty.java:85)
    at info.archinnov.achilles.internals.metamodel.UDTProperty.encodeFromJavaInternal(UDTProperty.java:51)
    at info.archinnov.achilles.internals.metamodel.AbstractProperty.encodeFromJava(AbstractProperty.java:57)
    at info.archinnov.achilles.internals.metamodel.AbstractProperty.encodeField(AbstractProperty.java:90)
    at info.archinnov.achilles.internals.runtime.BeanValueExtractor.lambda$extractAllValues$420(BeanValueExtractor.java:84)
    at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
    at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
    at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
    at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
    at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
    at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
    at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)
    at info.archinnov.achilles.internals.runtime.BeanValueExtractor.extractAllValues(BeanValueExtractor.java:86)
    at info.archinnov.achilles.internals.metamodel.AbstractEntityProperty.extractAllValuesFromEntity(AbstractEntityProperty.java:246)
    at info.archinnov.achilles.internals.dsl.crud.InsertWithOptions.getInternalBoundStatementWrapper(InsertWithOptions.java:137)
    at info.archinnov.achilles.internals.dsl.crud.InsertWithOptions.executeAsyncWithStats(InsertWithOptions.java:71)
    at info.archinnov.achilles.internals.dsl.action.MutationAction.executeWithStats(MutationAction.java:47)
    at com.prontoitlabs.mobiadz.repository.OfferRepository.save(OfferRepository.java:67)
    at com.prontoitlabs.mobiadz.service.impl.OfferServiceImpl.add(OfferServiceImpl.java:118)
    at com.prontoitlabs.mobiadz.controller.OfferController.add(OfferController.java:83)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory$1.invoke(ResourceMethodInvocationHandlerFactory.java:81)
    at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:144)
    at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:161)
    at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$ResponseOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:160)
    at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:99)
    at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:389)
    at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:347)
    at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:102)
    at org.glassfish.jersey.server.ServerRuntime$2.run(ServerRuntime.java:326)
    at org.glassfish.jersey.internal.Errors$1.call(Errors.java:271)
    at org.glassfish.jersey.internal.Errors$1.call(Errors.java:267)
    at org.glassfish.jersey.internal.Errors.process(Errors.java:315)
    at org.glassfish.jersey.internal.Errors.process(Errors.java:297)
    at org.glassfish.jersey.internal.Errors.process(Errors.java:267)
    at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:317)
    at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:305)
    at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:1154)
    at org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpContainer.service(GrizzlyHttpContainer.java:384)
    at org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:206)
    at org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:180)
    at org.glassfish.grizzly.http.server.HttpHandlerChain.doHandle(HttpHandlerChain.java:197)
    at org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:235)
    at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:119)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:283)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:200)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:132)
    at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:111)
    at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:77)
    at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:536)
    at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:112)
    at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:117)
    at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:56)
    at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:137)
    at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:591)
    at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:571)
    at java.lang.Thread.run(Thread.java:745)
```
Please help me to resolve this error, thanks.

**Comment by doanduyhai** [comment]
Which version of Achilles ??

**Comment by doanduyhai** [comment]
By the way, where is the `@PartitionKey` in the table `Dummy` ??? @Nikitakhanna92

**Comment by doanduyhai** [comment]
Ok issue successfully reproduced

**Comment by doanduyhai** [comment]
Fixed in Achilles 5.1.5
